Lesson 2/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        .myDiv{
```

Comment:
```suggestion
        .myDiv {
```
myDiv là cái gì đây?
rồi dấu cách đâu?